cmd/otelcol/config/collector/agent_config_linux.yaml

```diff
@@ -5,10 +5,12 @@ extensions:
   health_check:
   http_forwarder:
     egress:
-      endpoint: "https://api.${SPLUNK_REALM}.signalfx.com"
+      endpoint: "${SPLUNK_API_URL}"
   zpages:
 
 receivers:
+  fluentforward:
+    endpoint: 0.0.0.0:8006
```

Comment:
localhost is probably sufficient since the primary usecase is collecting local logs from fluentd.